terraform/modules/app/variables.tf

```diff
@@ -72,6 +72,12 @@ variable "load_balancers" {
   default     = []
 }
 
+variable "container_healthcheck_command" {
+  type        = list(string)
+  description = "App container readiness healthcheck"
```

Comment:
Did you mean liveness? (It looks like it's the Docker healthcheck, so that'd be roughly equivalent to a liveness probe in k8s, right?)

Reply:
@sengi thanks, I've fixed that 🙇